packages/jest-types/src/Config.ts

```diff
@@ -345,7 +345,7 @@ export type ProjectConfig = {
   displayName?: DisplayName;
   errorOnDeprecated: boolean;
   extensionsToTreatAsEsm: Array<Path>;
-  extraGlobals: Array<keyof NodeJS.Global>;
+  extraGlobals: Array<keyof typeof globalThis>;
```

Comment:
Do we know when `globalThis` was introduced to `@types/node`? Support was only added in Node.js 12 so in theory this could break `@types/node@10`. That version of Node.js is out of support but Jest still seems to include it in `package.json#/engines`.

Reply:
by default, `@types/node` 10 is installed in the project (when trigger yarn install)